File: openapi/code_samples/csharp/api_company_{cui}_financial_indicators_numeralFlux.cs
using Microsoft.AspNetCore.Mvc;
using Swashbuckle.AspNetCore.Annotations;

[ApiController]
[Route("api/[controller]")]
public class CompanyController : ControllerBase
{
    [HttpGet]
    [Route("numeral-flux/{cui}")]
    [SwaggerOperation(Summary = "Retrieves the numeral flux of a given company CUI.")]
    [ProducesResponseType(typeof(NumeralFluxReportResponseDTO), 200)]
    [ProducesResponseType(typeof(Fault), 400)]
    public IActionResult GetNumeralFlux([FromRoute] string cui, [FromQuery(Name = "currency")] string currency = "RON")
    {
        // Your implementation here
        return Ok();
    }
}

File: openapi/code_samples/javascript/api_company_{cui}_financial_indicators_numeralFlux.js
const express = require('express');
const app = express();
const port = 3000;

app.get('/api/company/numeral-flux/:cui', (req, res) => {
    const cui = req.params.cui;
    const currency = req.query.currency || 'RON';
    // Your implementation here
    res.status(200).json({});
});

app.listen(port, () => {
    console.log(`Server is listening at http://localhost:${port}`);
});
